### PR TITLE
A really long line

### PR DESCRIPTION
**Pull request by teoljungberg**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -2,4 +2,8 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  def index
+    render text: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
```
Comment (houndci-staging) [inline app/controllers/application_controller.rb:7]: Line is too long. [464/80]

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -6,3 +6,13 @@
 @import "neat";
 @import "base/base";
 @import "refills/flashes";
+
+a {
+  .b {
+    .c {
+      .d {
```
Comment (houndci-staging) [inline app/assets/stylesheets/application.scss:13]: Selector should have depth of applicability no greater than 2, but was 4

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -6,3 +6,13 @@
 @import "neat";
 @import "base/base";
 @import "refills/flashes";
+
+a {
+  .b {
+    .c {
```
Comment (houndci-staging) [inline app/assets/stylesheets/application.scss:12]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -6,3 +6,13 @@
...
+      .d {
+        color: blue;
```
Comment (houndci-staging) [inline app/assets/stylesheets/application.scss:14]: Color `blue` should be written in hexadecimal form as `#0000ff`<br>Color literals like `blue` should only be used in variable declarations; they should be referred to via variable everywhere else.

**Comment by teoljungberg** [comment]
Confirmed that https://github.com/thoughtbot/hound/pull/924 didn't break anything

**Pull request closed without merging**